### DISCUSSION TITLE
show_versions shows the PROJ version which pyproj was compiled against or the actual PROJ version used at runtime

### DISCUSSION THREAD

**Discussion by agiudiceandrea** [pyproj4/pyproj#1420] [Q&A] [answered]
It looks like the former. How is it possible to show the actual PROJ version used at runtime?

**Comment by snowman2** [comment]
It is intended to show the runtime version. Can you demonstrate why you believe that is not the case with a minimal reproducible example?

**Reply by agiudiceandrea** [reply]
Install QGIS 3.38.1 using the OSGeo4W installer https://www.qgis.org/resources/installation-guide/#osgeo4w-installer. Update proj_9_4.dll from version 9.4.0 provided by OSGeo4W to version 9.4.1 provided by conda-forge https://anaconda.org/conda-forge/proj/files?version=9.4.1

![image](https://github.com/user-attachments/assets/7b5f3e2b-e72d-4622-9299-5e0ad2c1ce74)

**Reply by snowman2** [reply, marked as answer]
Thanks! This should add similar information as QGIS is showing: https://github.com/pyproj4/pyproj/pull/1427

**Reply by agiudiceandrea** [reply]
Great! Thanks.
